package.json

```diff
@@ -1,73 +1,71 @@
 {
-  "name": "mpc-phase2-suite",
-  "description": "MPC Phase 2 suite of tools for conducting zkSNARKs trusted setup ceremonies",
-  "repository": "git@github.com:quadratic-funding/mpc-phase2-suite.git",
-  "homepage": "https://github.com/quadratic-funding/mpc-phase2-suite",
-  "bugs": "https://github.com/quadratic-funding/mpc-phase2-suite/issues",
-  "license": "MIT",
-  "private": true,
-  "workspaces": {
-    "packages": [
-      "phase2cli",
-      "firebase"
+    "name": "zkmpc",
+    "description": "MPC Phase 2 suite of tools for conducting zkSNARKs trusted setup ceremonies",
+    "repository": "git@github.com:quadratic-funding/mpc-phase2-suite.git",
+    "homepage": "https://github.com/quadratic-funding/mpc-phase2-suite",
+    "bugs": "https://github.com/quadratic-funding/mpc-phase2-suite/issues",
+    "license": "MIT",
+    "private": true,
+    "keywords": [
+        "typescript",
+        "zero-knowledge",
+        "zk-snarks",
+        "phase-2",
+        "trusted-setup",
+        "ceremony",
+        "snarkjs",
+        "circom"
     ],
-    "nohoist": [
-      "**/snarkjs",
-      "**/snarkjs/**"
+    "scripts": {
+        "build": "lerna run build",
+        "pretest": "yarn build",
```

Comment:
Duplicated of `build`

Reply:
I was thinking of using the `pretest` npm script to launch a build (via `yarn build` script) before the actual test execution. So, the `pretest` is executed when running `test` script while `build` can be thought as standalone npm script for building.

Reply:
Then it's a kind of subtask of `test` script, then something like `test:prepare` might be better.

Reply:
I understand but the `pretest` naming convention is officially used by npm scripts (reserved keyword for scripts). In fact, the `pretest` script is automatically executed when running the `test` script. See here (https://docs.npmjs.com/cli/v9/using-npm/scripts) @gurrpi

Reply:
Fair enough. Didn't know that it is part of life cycle scripts.

Reply:
No worries :) <3